src/showcase/Overview.vue

```diff
@@ -85,11 +85,16 @@
             <FocusTrapZoneExample/>
         </OverviewItem>
 
+        <OverviewItem title="Overlay">
+            <OfficeToggle onText="Hide the Overlay" offText="Show the Overlay" v-model="showOverlay">Test</OfficeToggle>
+            <OfficeOverlay v-if="showOverlay" @click.native="showOverlay = false">
```

Comment:
Use `v-show` here, as it will be toggled multiple times. Let's see if it still works then, as `v-show` does only set `display: none` and does not trigger the `beforeDestroy` lifecycle hook and will therefore probably mess up the body scroll even if it's not visible. You might have to add a Prop to the `OfficeOverlay` like `visible`, which toggles the visibility. Then you can watch this property and enable/disable body scroll correctly.

Reply:
testing e213ad0f98b4388b8a2782f9030050eafb4d1027